pom.xml

```diff
@@ -16,8 +16,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
+            <groupId>junit</groupId>
```

Comment:
This by itself won't work. You need to change it in `<dependencyManagement>` section as well. Here is where you can find all the necessary information about this artifact: https://mvnrepository.com/artifact/junit/junit